picard/cluster.py

```diff
@@ -499,7 +531,15 @@ def cluster(self, threshold):
                     c = similarity(token_x, token_y)
                     if c >= threshold:
                         heappush(heap, ((1.0 - c), [x, y]))
-            QtCore.QCoreApplication.processEvents()
+            if tagger and (y % status_update_steps) == 0:
+                statusmsg = _("Clustering - step %(step)d/3: %(cluster_type)s (%(update)d%%)")
+                mparams = {
+                    'step': self.cluster_type.value,
+                    'cluster_type': _(self._cluster_type_label()),
+                    'update': 5 * y // status_update_steps,
```

Comment:
A minor issue, got 110% :)

```
2020-06-17 10:59:48,137 - Clustering - step 1/3: Metadata Extraction (110%)
```

Reply:
Haha, that is unexpected. I've seen 105%, but thought it was due to rounding error

Reply:
Oh, I think I get it. Must be due to status_update_steps == 1 when the number of files < 10 or 5.

Reply:
It isn't the most elegant solution, nor guarantee the 100%, but 110% is no more.